R/ppc-loo.R

```diff
@@ -167,7 +167,8 @@ ppc_loo_pit_overlay <- function(y,
   }
 
   if (boundary_correction) {
```

Comment:
Maybe move the NOTE message out of this if statement? It hold also for the old kernel density estimate.